src/ngraph/pass/opset0_downgrade.cpp

```diff
@@ -181,22 +181,37 @@ bool pass::Opset0Downgrade::run_on_node(shared_ptr<Node> node)
     case OP_TYPEID::ConvolutionBackpropData_v1:
     {
         auto tmp = as_type_ptr<op::v1::ConvolutionBackpropData>(node);
-        NGRAPH_CHECK(node->input_value(2).get_node_shared_ptr()->is_constant());
-        auto data_batch_shape =
-            static_pointer_cast<op::Constant>(node->input_value(2).get_node_shared_ptr())
-                ->get_shape_val();
-        const auto filters_arg = node->input(0).get_source_output();
-        const auto delta_arg = node->input(1).get_source_output();
+        auto output_shape = as_type_ptr<op::Constant>(node->input_value(2).get_node_shared_ptr());
+        const auto data_arg = node->input(0).get_source_output();
+        const auto filters_arg = node->input(1).get_source_output();
         const PartialShape& delta_arg_pshape = node->get_input_partial_shape(1);
         NGRAPH_CHECK(delta_arg_pshape.rank().is_static(),
                      "Unable to convert ConvolutionBackpropData:v1 to ConvolutionBackpropData:v0 "
                      "if delta argument rank is dynamic. Node: ",
                      *node);
+        NGRAPH_CHECK(output_shape,
+                     "Unable to convert ConvolutionBackpropData:v1 to ConvolutionBackpropData:v0 "
+                     "if output_shape is not constant. Node: ",
+                     *node);
         const size_t num_spatial_dims = static_cast<size_t>(delta_arg_pshape.rank()) - 2;
+
+        auto output_padding = tmp->get_output_padding();
+
+        bool is_op_valid = true;
+        for (auto value : output_padding)
+        {
+            is_op_valid = is_op_valid && (value == 0);
```

Comment:
TBH I don't like this check very much. Can you use `std::all_of` instead with a lambda that checks the equality of elements with zero instead?

Reply:
Done.